policyengine_us/parameters/gov/states/mi/tax/income/deductions/retirement_benefits/tier_one/birth_year.yaml

```diff
@@ -0,0 +1,20 @@
+description: Michigan provides a retirement and pension benefits addition for tier 1 recipients born before this year. 
+values:
+  2017-01-01: 1941
+  2018-01-01: 1942
+  2019-01-01: 1943
+  2020-01-01: 1944
+  2021-01-01: 1945
+  2022-01-01: 1946
+
+metadata:
+  unit: year
+  reference:
+    - title: Michigan Legal Code Section 206.30 - (8)
```

Comment:
Where in this legal code subsection is this specified? I only skimmed it but seems like it could be possible to simplify the logic since e.g. `1942` doesn't show up.